### PR TITLE
Convert opscode.com and getchef.com to chef.io in README

### PR DESCRIPTION
**Pull request by danielsdeleo**
Didn't see an open PR for this in the first few pages.

@chef/client-engineers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,11 +64,11 @@ Then get the source and install it:
 
 Before working on the code, if you plan to contribute your changes, you need to
 read the
-[Chef Contributions document](http://docs.opscode.com/community_contributions.html).
+[Chef Contributions document](http://docs.chef.io/community_contributions.html).
 
 You will also need to set up the repository with the appropriate branches. We
 document the process on the
-[Working with Git](http://wiki.opscode.com/display/chef/Working+with+git) page
+[Working with Git](http://wiki.chef.io/display/chef/Working+with+git) page
```
Comment (danielsdeleo) [inline README.md]: Wiki is dead, what should we do here? Just delete the section? Is there a github doc we should link to?

Comment (lamont-granquist) [inline README.md]: I bet that information was from 3 years and out of date anyway.  Something along the lines of "just make a pull request against master" is the current process for external contributors.

File: README.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ Chef - A configuration management system
 
 |                      |                                          |
 |:---------------------|:-----------------------------------------|
-| **Author:**          | Adam Jacob (<adam@opscode.com>)
+| **Author:**          | Adam Jacob (<adam@chef.io>)
 | **Copyright:**       | Copyright (c) 2008-2014 Chef Software, Inc.
```
Comment (lamont-granquist) [inline README.md]: ^ 2015 if you're *really* feeling OCD

File: README.md [hunk 3]
```diff
@@ -17,8 +17,8 @@ entire infrastructure.
 This README focuses on developers who want to modify Chef source code.
 If you just want to use Chef, check out these resources:
 
-* [learnchef](https://learnchef.opscode.com): Getting started guide
-* [http://docs.opscode.com](http://docs.opscode.com): Comprehensive User Docs
+* [learnchef](https://learn.chef.io): Getting started guide
+* [http://docs.chef.io](http://docs.chef.io): Comprehensive User Docs
 * [Installer Downloads](http://www.getchef.com/chef/install/): Install Chef as a complete package
```
Comment (drag00n) [inline README.md]: Should this be https://downloads.chef.io/index.html ?

**Comment by lamont-granquist** [comment]
:shipit:

**Pull request merged**